openzwavemqtt/models/value.py

```diff
@@ -64,16 +63,9 @@ def command_class(self) -> CommandClass:
         return self.parent.command_class_id
 
     @property
-    def index(self) -> ValueIndex:
+    def index(self) -> int:
         """Return Index."""
-        try:
-            # TODO: we can make this prettier by returning the
-            # ValueIndex belonging to this specific CC
-            # Now it will return the first value matching this int value
-            # Nothing breaks but it isn't very pretty
-            return ValueIndex(self.data.get("Index"))
-        except ValueError:
-            return ValueIndex.UNKNOWN
+        return self.data.get("Index")
```

Comment:
Do we know that the `"Index"` key is always present in data? Then we could use `dict[key]`.

Reply:
Good point, by the time the value gets added to mqtt it should have the index available. Updated the code.